acados/ocp_nlp/ocp_nlp_ddp.c

```diff
@@ -1046,7 +1035,7 @@ void ocp_nlp_ddp_eval_param_sens(void *config_, void *dims_, void *opts_, void *
     ocp_nlp_common_eval_param_sens(config, dims, opts->nlp_opts, nlp_mem, nlp_work,
                                  field, stage, index, sens_nlp_out);
 
-    mem->time_solution_sensitivities = acados_toc(&timer0);
+    nlp_mem->nlp_timings->time_solution_sensitivities = acados_toc(&timer0);
```

Comment:
only `timings`?

Reply:
Here I kept it, thought it is not worth to unpack the pointer if only one field is used.
Note that this is not in the main ddp function.